cars/car.js

```diff
@@ -1,9 +1,18 @@
 module.exports = Car
 
-function Car(mpg) {
-
+function Car({mpg, odometer, trips, gallons}) {
+if (!(this instanceof Car)) { return new Car({mpg, odometer, trips, gallons}) }
```

Comment:
👍 on the `instanceof` magic.

But why did you change the API of `Car`? Looks like it only accepts a single arg `mpg`